src/dbnode/storage/stats/query_stats_default_tracker.go

```diff
@@ -31,31 +32,71 @@ import (
 const defaultLookback = time.Second * 5
 
 // Tracker implementation that emits query stats as metrics.
-type queryStatsMetricsTracker struct {
+type queryStatsTracker struct {
 	recentDocs tally.Gauge
 	totalDocs  tally.Counter
+
+	config QueryStatsConfig
+}
+
+var _ QueryStatsTracker = (*queryStatsTracker)(nil)
+
+// DefaultQueryStatsConfigForMetrics returns a default
+// config for tracking query stats as metrics.
+func DefaultQueryStatsConfigForMetrics() QueryStatsConfig {
+	return QueryStatsConfig{
+		MaxDocs:  nil,
+		Lookback: defaultLookback,
+	}
 }
 
-var _ QueryStatsTracker = (*queryStatsMetricsTracker)(nil)
+// DefaultQueryStatsConfigForMetricsAndLimits returns a
+// default config for tracking query stats as metrics.
+func DefaultQueryStatsConfigForMetricsAndLimits(
```

Comment:
nit: since it's a public struct with no special construction going on, are these methods necessary?

Reply:
Agreed, if we make "MaxDocs" a non pointer type, we can also rename this struct to "QueryStatsOptions" and just have it as a "by value" options struct.